Microsoft.NET.Build.Containers/ParseContainerProperties.cs

```diff
@@ -106,11 +101,29 @@ public override bool Execute()
             return !Log.HasLoggedErrors;
         }
 
+        try
+        {
+            if (!ContainerHelpers.NormalizeImageName(ContainerImageName, out string? normalizedImageName))
+            {
+                Log.LogWarning($"{nameof(ContainerImageName)} was not a valid container image name, it was normalized to {normalizedImageName}");
```

Comment:
Yeah, we should definitely do this. Something like `OCI001`? Unsure what prefix we should use. Any guidelines?

Reply:
I added a code that probably won't be what we end up on, but can at least be a discussion point.

Reply:
I think this is fine for now; we'll probably converge on a `NETSDK` prefix like everything(?) in the SDK repo but this will keep us from accidentally stepping on that.